resources/sql_scripts/CVManager_CreateTables.sql

```diff
@@ -543,4 +543,24 @@ CREATE TABLE IF NOT EXISTS public.max_retry_limit_reached_instances
       REFERENCES public.firmware_images (firmware_id) MATCH SIMPLE
       ON UPDATE NO ACTION
       ON DELETE NO ACTION
-);
\ No newline at end of file
+);
+
+
+
+-- Indexes
+CREATE INDEX idx_organizations_name ON public.organizations (name);

```

Comment:
Neat idea to make indexes. I hadn't thought about indexes in a smaller database. Is this a noticeable difference in performance or precautionary?

Reply:
just precautionary, the api will now send off "check if rsu in org" calls with some requests, don't want that to bog the system down